distribution/Makefile

```diff
@@ -52,7 +52,10 @@ publish: container
 	docker push ${IMAGE_NAME}:latest
 	$(MAKE) -C environments $@
 
-run:
+create-squid-cache-volume:
+	docker volume create squid-cache
+
+run: create-squid-cache-volume
 	$(COMPOSE) up
 
 clean:
```

Comment:
Please add some clean up for the volume to `make clean`

Reply:
done